tests/test_main.py

```diff
@@ -337,3 +337,44 @@ class Config:
     with pytest.raises(ValueError) as exc_info:
         m.b = 11
     assert '"TestModel" object has no field "b"' in str(exc_info)
+
+
+def test_validating_assignment():
+    import pydantic
+
+    class Pedantic(BaseModel):
```

Comment:
use `ValidateAssignmentModel` as the name here and move outside the test so it can be used in multiple tests.